examples/2d/elixir_euler_unstructured_quad.jl

```diff
@@ -0,0 +1,54 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_convergence_test
+source_terms = source_terms_convergence_test
+boundary_conditions = boundary_condition_convergence_test
+
+###############################################################################
+# Get the DG approximation space
+
+polydeg = 5
+surface_flux = flux_hll
+solver = DGSEM(polydeg, surface_flux)
+
+###############################################################################
+# Get the curved quad mesh from a file
+
+mesh_file = joinpath(@__DIR__, "BoxAroundCircle8.mesh")
```

Comment:
okay, I will change this in the name of consistency, plus it will make it a fun surprise for people when they visualize the result and actually see what mesh they just used.

Reply:
Although I just realized working on this that such naming is weird. It makes it sound like the mesh would only work for Euler (or am I just crazy?)

Reply:
That's a good point. Since we prefer descriptive variable names, shouldn't we also prefer descriptive mesh file names?

Reply:
You're right. Maybe just prefix their names with `mesh_` (as opposed to elixir) and use `snake_case`, e.g.,
```
mesh_box_around_circle.mesh
mesh_periodic_square_with_twist.mesh
```
or something similar? I'd leave out the polydeg unless we have several meshes with the same name and different degree (which I doubt we will, at least not in the `examples/` folder).